### DISCUSSION TITLE
clipboard writeText api only string is supported, we also need to support other formats of parameters

### DISCUSSION THREAD

**Discussion by wkl007** [tauri-apps/tauri#4888] [Ideas]
like electron 

https://www.electronjs.org/docs/latest/api/clipboard#clipboardwritedata-type

const { clipboard } = require('electron')

clipboard.write({
  text: 'test',
  html: '<b>Hi</b>',
  rtf: '{\\rtf1\\utf8 text}',
  bookmark: 'a title'
})

console.log(clipboard.readText())
// 'test'

console.log(clipboard.readHTML())
// <meta charset='utf-8'><b>Hi</b>

console.log(clipboard.readRTF())
// '{\\rtf1\\utf8 text}'

console.log(clipboard.readBookmark())
// { title: 'a title', url: 'test' }

**Comment by FabianLars** [comment]
Can you add a comment with the formats you need in issue https://github.com/tauri-apps/tauri/issues/2786 please?

**Reply by wkl007** [reply]
Sorry i can't do it, i'm just a rust newbie.
